.eslintrc.yml

```diff
@@ -114,7 +118,7 @@ rules:
   '@typescript-eslint/no-var-requires': error
 
   # These should be disabled by eslint-config-prettier, but are not.
-  spaced-comment: 'off'
+  no-extra-semi: 'off'
```

Comment:
I'm a little confused by this rule swap here. Do we no longer need `spaced-comment` disabled?

Reply:
Ah! Good question.

Without `no-extra-semi: 'off'` there are a bunch of errors related to things Prettier wants just so.

Without `spaced-comment: 'off'` there are no errors, which makes me think that it's no longer needed.

Reply:
Gotcha, thanks!